.github/workflows/system-tests-enterprise.yml

```diff
@@ -83,39 +85,37 @@ jobs:
           automation/system_test/dev_utilities.py \
           ${{ secrets.LATEST_SYSTEM_TEST_DATA_CLUSTER_SSH_USERNAME }}@${{ secrets.LATEST_SYSTEM_TEST_DATA_CLUSTER_IP }}:/home/iguazio/dev_utilities.py
 
-        sshpass \
-          -p "${{ secrets.LATEST_SYSTEM_TEST_DATA_CLUSTER_SSH_PASSWORD }}" \
-          ssh \
-          -o StrictHostKeyChecking=no \
-          -o ServerAliveInterval=180 \
-          -o ServerAliveCountMax=3 \
-          ${{ secrets.LATEST_SYSTEM_TEST_DATA_CLUSTER_SSH_USERNAME }}@${{ secrets.LATEST_SYSTEM_TEST_DATA_CLUSTER_IP }} \
-            LC_ALL=en_US.utf8 LANG=en_US.utf8 /bin/python3 \
-              /home/iguazio/cleanup.py \
-              docker-images \
-              http://localhost:8009 \
-              igz0.docker_registry.0 \
-              "ghcr.io/mlrun/mlrun-api,ghcr.io/mlrun/mlrun-ui,ghcr.io/mlrun/mlrun,ghcr.io/mlrun/ml-base,ghcr.io/mlrun/log-collector"
-
-
   prepare-system-tests-enterprise-ci:
     # When increasing the timeout make sure it's not larger than the schedule cron interval
     timeout-minutes: 55
     name: Prepare System Tests Enterprise
-    runs-on: ubuntu-latest
-
+    runs-on: [ self-hosted, Linux ]
+    container:
+      image: python:3.9
     needs: [system-test-cleanup]
     # let's not run this on every fork, change to your fork when developing
     if: github.repository == 'mlrun/mlrun' || github.event_name == 'workflow_dispatch'
 
     steps:
     - uses: actions/checkout@v3
+    - name: Install dependencies
+      run: |
+        apt-get update -qqy && apt-get install -y sshpass jq curl gnupg nodejs
```

Comment:
I'm guessing nodejs is needed for some specific tests?

Reply:
to run some github actions steps